vireoSDK.props

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <PropertyGroup>
+        <vireo_InstallDir>$(MSBuildThisFileDirectory)..\content\</vireo_InstallDir>
+    </PropertyGroup>
+</Project>
```

Comment:
Add a newline to the end of the file to remove the red artifact that appears in online version of the diff at the end of the file.

Reply:
Done.